interpreter/namerd/src/main/scala/io/buoyant/namerd/iface/NamerdInterpreterInitializer.scala

```diff
@@ -107,7 +107,7 @@ case class NamerdInterpreterConfig(
     val tlsParams = tls.map(_.params).getOrElse(Stack.Params.empty)
 
     val client = ThriftMux.client
-      .withParams(ThriftMux.client.params ++ tlsParams ++ params + Thrift.ThriftImpl.Netty4)
+      .withParams(ThriftMux.client.params ++ tlsParams ++ params ++ Thrift.client.params)
```

Comment:
I think that since the netty3 implementation has gone away, we can simply remove `Thrift.ThriftImpl.Netty4` entirely.  I don't think we want to replace it with `Thrift.client.params`. (unless there's something I'm missing?)